### DISCUSSION TITLE
Craft type attribute

### DISCUSSION THREAD

**Discussion by frankbarry** [etsy/open-api#685] [Q&A] [answered]
For certain categories, a "Craft type" field appears in the shop manager interface that can be configured with values such as "Baking", "Basket weaving" etc. I don't see a way to set this using the API. Did I miss something or is this a manual setting only?

**Comment by etsyachristensen** [comment, marked as answer]
@barryfrank The "Craft type" is a property attribute that is available on some taxonomies. The taxonomy will determine which attribute values are available to use.

For example, when you use taxonomy id 2310 (Home & Living > Kitchen & Dining > Bakeware > Bundt Pans) you have the options of:
"Baking" and "Kid's crafts".

If you call to `getPropertiesByTaxonomyId` with that taxonomy id, you'll see the possible properties and their values.
